app/admin/api/v3/top_profile.rb

```diff
@@ -5,8 +5,13 @@
 
   # before creating add summary, year and profile_type to top profile record
   before_create :derive_top_profile_details
+  after_action :clear_cache, only: [:create, :update, :destroy]
 
   controller do
+    def clear_cache
+      clear_cache_for_regexp('/api/v3/contexts')
```

Comment:
I think the url in this case is different: `/api/v3/top_profiles`?

Reply:
🤦‍♀